libs/books/feature/src/lib/book-search/book-search.component.html

```diff
@@ -6,8 +6,9 @@
       type="search"
       placeholder="Search for books to add to your reading list"
       formControlName="term"
+      aria-label="searchBar"
     />
-    <button mat-icon-button matSuffix>
+    <button aria-label="seachButton" mat-icon-button matSuffix>
```

Comment:
Do you think this would sound correct to a screen reader? What about simply "Search Button"?